org.eclipse.lsp4e.tests.mock/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Mock Language Server to test LSP4E
 Bundle-SymbolicName: org.eclipse.lsp4e.tests.mock
-Bundle-Version: 0.14.4.qualifier
+Bundle-Version: 0.14.5.qualifier
```

Comment:
This shouldn't be necessary. Please revert and consider the version bump separately if really needed.

Reply:
I cannot reproduce a failure with a local build. Please submit the PR without changing versions to see how it does behave on CI.